bulk_lookup/forms.py

```diff
@@ -0,0 +1,116 @@
+import logging
+
+from django import forms
+from django.conf import settings
+from django.utils.crypto import get_random_string
+
+import stripe
+from ukpostcodeutils.validation import is_valid_postcode
+
+from .models import OutputOption
+
+
+logger = logging.getLogger(__name__)
```

Comment:
There's no log handler set up for `bulk_lookup` so logged messages won't go anywhere.